File: PRAK304-2210817120007-NursyahnaPuteri.c
#include <stdio.h>
int main ()
{
    int bilangan ;

    printf ("Masukan Bilangan : ");
    scanf ("%d" , &bilangan);

    if (bilangan >= 1 && bilangan <= 9) {
        printf ("Satuan\n");
   }else if ( bilangan >=10 && bilangan <= 19){
        printf ("Belasan\n");
   }else if ( bilangan >= 20 && bilangan <= 99) {
        printf ("Puluhan\n" ) ;
   }else if ( bilangan == 0 ) {
        printf ( "Nol\n");
   }else {
        printf ("Anda Menginput Melebihi Limit Bilangan\n");
     }
     printf ("\n");
     return 0;
} 

File: PRAK304-2210817120007-NursyahnaPuteri.py
bilangan = int (input ("Masukan Nilai : "))

if bilangan >= 1 and bilangan <=9 :
    print ("Satuan")
elif bilangan >=10 and bilangan <=19:
    print ("Belasan")
elif bilangan >=20 and bilangan <=99:
    print ("Puluhan")
elif bilangan == 0 :
    print ("Nol")
else :
    print ("Anda Menginput Melebihi Limit Bilangan")